src/query/storages/parquet/src/table_function/table.rs

```diff
@@ -67,10 +78,31 @@ impl ParquetTable {
                 "Should enable `allow_insecure` to use table function `read_parquet`",
             ));
         }
+        let mut builder = opendal::services::fs::Builder::default();
+        builder.root("/");
+        let operator = Operator::new(builder.build()?);
+
+        Self::create(
+            database_name,
+            table_func_name,
+            table_id,
+            table_args,
+            operator,
+        )
+    }
 
-        // Syntax:
-        // read_parquet('path1', 'path2', ..., prune_pages=>true, refresh_meta_cache=>true, ...)
-        // The options should be behind the file paths.
+    pub fn create(
+        database_name: &str,
+        table_func_name: &str,
+        table_id: u64,
+        table_args: TableArgs,
+        operator: Operator,
+    ) -> Result<Arc<dyn TableFunction>> {
+        if !GlobalConfig::instance().storage.allow_insecure {
+            return Err(ErrorCode::StorageInsecure(
+                "Should enable `allow_insecure` to use table function `read_parquet`",
```

Comment:
I guess the `create` is to be called by `select from stage`, (I`m trying something like this locally)

1. should not check it in `create_table_function` here?
2. instead of faking a  table_args outside, what about this


```
    pub fn create_from_files(
        database_name: &str,
        table_func_name: &str,
        table_id: u64,
        file_locations: Vec<String>,
        operator: Operator,
        read_options: ReadOptions,
        table_args: Vec<Scalar>,
    ) -> Result<Arc<dyn TableFunction>> 
```
```

Reply:
I will add other methods for `select from stage`.